framework/src/org/checkerframework/common/value/javadoc.astub

```diff
@@ -0,0 +1,7 @@
+import org.checkerframework.common.value.qual.MinLen;
+
+package com.sun.javadoc;
+
+interface RootDoc {
+    String @MinLen(1) [][] options();
```

Comment:
Should this be `String [] @MinLen(1) []`?
https://docs.oracle.com/javase/8/docs/jdk/api/javadoc/doclet/com/sun/javadoc/RootDoc.html#options--
doesn't indicate that there always are options. The classdoc says `Also represents the command line information -- the packages, classes and options specified by the user.` which seems to suggest that if I only call `javadoc mypackage` then options is empty?
But for each option, the second dimension is always at least 1 element long.

Reply:
Thanks for catching the misplaced annotation.  I fixed this.